lms/envs/common.py

```diff
@@ -2870,3 +2870,14 @@
 TYPES_WITH_CHILD_PROBLEMS_LIST = [
     'library_content',
 ]
+
+# These are the problem types (by block_type) that an instructor can query over in the Instructor Query tab
+INSTRUCTOR_QUERY_PROBLEM_TYPES = [
```

Comment:
You'll still need the corresponding entry in `aws.py`,
else your configuration/json will never be read in ;)

```python
INSTRUCTOR_QUERY_PROBLEM_TYPES = ENV_TOKENS.get('INSTRUCTOR_QUERY_PROBLEM_TYPES', INSTRUCTOR_QUERY_PROBLEM_TYPES)
```